.github/workflows/ci.yml

```
@@ -43,6 +43,7 @@ jobs:
           instrumentation-grpcnetclient: ['*/OpenTelemetry.Instrumentation.GrpcNetClient*/**', '!**/*.md']
           instrumentation-hangfire: ['*/OpenTelemetry.Instrumentation.Hangfire*/**', '!**/*.md']
           instrumentation-http: ['*/OpenTelemetry.Instrumentation.Http*/**', '!**/*.md']
+          instrumentation-kafka: ['*/OpenTelemetry.Instrumentation.ConfluentKafka*/**', '!**/*.md']
```

Please use full name of the package, without opentelemetry. prefix.
Also include examples folder

```suggestion
          instrumentation-confluentkafka: ['*/OpenTelemetry.Instrumentation.ConfluentKafka*/**', 'examples/redis/**',  '!**/*.md']
```

Alphabetical reordering + changes in other places in this file is also needed.

---

done